templates/takeovers/_fips_certification_cis_compliance.html

```diff
@@ -0,0 +1,14 @@
+{% with title="FIPS certification and CIS compliance with Ubuntu",
+subtitle="Learn about Ubuntu CIS and FIPS certified components to enable operating environments under compliance regimes like FedRAMP, HIPAA, PCI and ISO.",
+header_image="https://assets.ubuntu.com/v1/b576398c-compliance-wht.svg",
+image_width="400",
+image_height="234",
+image_hide_small="true",
+equal_cols="true",
+primary_url="https://www.brighttalk.com/webcast/6793/432536?utm_source=Takeover",
+primary_cta="Register",
```

Comment:
```suggestion
primary_cta="Register for the webinar",
```

Sorry, I thought Kelly was changing it.